corehq/apps/callcenter/tasks.py

```diff
@@ -58,12 +69,18 @@ def sync_usercases_if_applicable(user, spawn_task):
         return
     if (user.project.call_center_config.enabled or user.project.usercase_enabled):
         if spawn_task:
-            sync_usercases_task.delay(user._id)
+            sync_usercases_task.delay(user._id, user.project)
```

Comment:
Similar to above, pass in the domain name rather than the whole domain:
```suggestion
            sync_usercases_task.delay(user._id, user.domain)
```